src/test/java/com/googleinterns/zoomtube/mocks/MockResponse.java

```diff
@@ -0,0 +1,63 @@
+package com.googleinterns.zoomtube.mocks;
+
+import static org.mockito.Mockito.mock;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletResponseWrapper;
+
+/**
+ * This mocks {@link javax.servlet.http.HttpServletResponse} to allow for testing
+ * servlets. As more tests and servlets are added, you may need to override additional
```

Comment:
nit: Having "you" in the javaDoc is a bit weird to me personally. Can you rephrase it to say something like 
"As more tests and servlets are added, additional methods may need to be overridden."? (This also shows up in MockRequest)